src/zc/table/column.py

```diff
@@ -57,12 +67,12 @@ def _sort(self, items, formatter, start, stop, sorters, multiplier):
         if self.subsort and sorters:
             items = sorters[0](items, formatter, start, stop, sorters[1:])
         else:
-            items = list(items) # don't mutate original
+            items = list(items)  # don't mutate original
         getSortKey = self.getSortKey
 
         items.sort(
-            cmp=lambda a, b: multiplier*cmp(a, b),
-            key=lambda item: getSortKey(item, formatter))
+            key=lambda item: getSortKey(item, formatter),
+            reverse=(multiplier == -1))
```

Comment:
If `multiplier` is not part of the external API, I'd suggest refactoring it so we're passing a `reverse` internally.

Reply:
_sort() is called exactly twice, with multiplier passed as a constant value (1 or -1).

Please change the signature to take a reverse instead of multiplier, and the two call sites to pass reverse=False/True, (perhaps as a keyword argument for readability).

Reply:
done